avocado/core/nrunner.py

```diff
@@ -696,7 +696,8 @@ async def cb(self, reader, _):
                     return True
 
             message = await reader.readline()
-            if message == b'bye\n':
+            exit_command = message.splitlines()
```

Comment:
`message` is already a line, as it's returned from `readline()`, so it's a bit strange to use `splitlines()` here IMO.  Like I said before, a `strip()` should be able to the job IMO:

```
In [1]: b'bye\r\n'.strip()                                                                                                                        
Out[1]: b'bye'

In [2]: b'bye\r'.strip()                                                                                                                          
Out[2]: b'bye'

In [3]: b'bye\n'.strip()                                                                                                                          
Out[3]: b'bye'
```

Reply:
Indeed, I read your comment in the previous review and for some reason, I had `split` in mind instead of `strip`. Sending a new version. Thanks!